Tests/BuiltInRulesTests/NestingRuleTests.swift

```diff
@@ -555,4 +555,48 @@ final class NestingRuleTests: SwiftLintTestCase {
 
         verifyRule(description, ruleConfiguration: ["ignore_typealiases_and_associatedtypes": true])
     }
+
+    func testNestingWithoutCodingKeys() {
```

Comment:
You can move these examples directly into `NestingRuleExamples`. Any options deviating from the default will be added to the online documentation, so it's clear why an example triggers or why it doesn't.

Reply:
In the documentation that includes these examples later, we now also show the configuration in case it deviates from the default one. So having them as normal examples is even nicer as people also see how options influence the rule's behavior.

Reply:
The feature that configurations are added to the examples in the documentation is quite new and we won't convert existing tests. But we should prefer this way of documenting examples from now on.